docs/contributing/API Review Process.md

```diff
@@ -0,0 +1,46 @@
+    * **Close as not actionable**. In case the issue didn't get enough traction to be distilled into a concrete proposal, the owner will close the issue.
+    * **Close as won't fix as proposed**. Sometimes, the issue that is raised is a good one but the owner thinks the concrete proposal is not the right way to tackle the problem. In most cases, the owner will try to steer the discussion in a direction that results in a design that we believe is appropriate. However, for some proposals the problem is at the heart of the design which can't easily be changed without starting a new proposal. In those cases, the owner will close the issue and explain the issue the design has.
+    * **Close as won't fix**. Similarly, if proposal is taking the product in a direction we simply don't want to go, the issue might also get closed. In that case, the problem isn't the proposed design but in the issue itself.
+
+5. **API gets reviewed**. The group conducting the review is called *RAR*, which stands for *Roslyn API Reviewers*. In the review, we'll take notes and provide feedback. After the review, we'll publish the notes in the [API Review repository](https://github.com/dotnet/apireviews) and at the end of the relevant issue. Multiple outcomes are possible:
+
+    * **Approved**. In this case the label `api-ready-for-review` is replaced with `api-approved`.
+    * **Needs work**. If we believe the proposal isn't ready yet, we'll replace the label `api-ready-for-review` with `api-needs-work`.
+    * **Rejected**. If we believe the proposal isn't a direction we want to pursue, we'll simply write a comment and close the issue.
+
+## Review schedule
+
+ There are three methods to get an API review (this section applies to API area owners, but is included for transparency into the process):
+
+* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/roslyn` and applying the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while. Progress of issues can be tracked via <https://apireview.net/backlog?g=Roslyn>.
+* **Fast track**. If you need to bypass the backlog apply both `api-ready-for-review` and `blocking`. All blocking issues are looked at before we walk the backlog.
+* **Dedicated review**. If an issue you are the area owner for needs an hour or longer, send an email to roslynapiowners and we book dedicated time. We also book dedicated time for language feature APIs: any APIs added as part of a new language feature need to have a dedicated review session. Rule of thumb: if the API proposal has more than a dozen APIs, the APIs have complex policy, or the APIs are part of a feature branch, then you need 60 min or more. When in doubt, send mail to roslynapiowners.
+
+The API review board currently meets biweekly.
```

Comment:
Is this `biweekly` twice a week, or once every two weeks?

Reply:
Biweekly is always once every two weeks. Semiweekly is twice a week.

Reply:
Is that in a style guide? It's ambiguous in normal English. 😄

Reply:
> Is that in a style guide? It's ambiguous in normal English. smile

It's how I've always interpreted it, certainly. I'll adjust the wording to be a bit more verbose.
```suggestion
The API review board currently meets every 2 weeks.
```

Reply:
> It's how I've always interpreted it, certainly.

It's prescriptively correct, but as a reader of the doc alone, I don't know if it's intending it that way or if it might be "wrong". (If there were a `semi-` nearby, could infer.) Thanks.

Reply:
I avoid the word. https://www.merriam-webster.com/dictionary/biweekly:

1. occurring every two weeks : fortnightly
2. occurring twice a week